src/styles/blocks/calendar.scss

```
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$dayGap};
+  width: getWidth($daySize, $dayGap);
+  margin: 10px auto;
+
+  @each $dayName, $number in $days {
+    &--start-day--#{$dayName}
+    &__day:first-child {
+      margin-left: ($number * $daySize + $number * $dayGap);
+    }
+  }
+
+  @for $lastDay from 28 through 31 {
+    &--month-length--#{$lastDay}
+    &__day:nth-child(n + #{$lastDay + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include square(#{$daySize});
+
+    border: 1px solid black;
+    background-color: #eee;
+    transition: background-color 0.5s, transform 0.5s;
```

It's not important, but you can add `transition-timing-function` and change order
```suggestion
    transition: 0.5s background-color ease, 0.5s transform ease;
```